server/public/model/integration_action.go

```diff
@@ -523,10 +536,21 @@ func (e *DialogElement) IsValid() error {
 	case "select":
 		multiErr = multierror.Append(multiErr, checkMaxLength("Default", e.Default, DialogElementSelectMaxLength))
 		multiErr = multierror.Append(multiErr, checkMaxLength("Placeholder", e.Placeholder, DialogElementSelectMaxLength))
-		if e.DataSource != "" && e.DataSource != "users" && e.DataSource != "channels" {
-			multiErr = multierror.Append(multiErr, errors.Errorf("invalid data source %q, allowed are 'users' or 'channels'", e.DataSource))
+		if e.DataSource != "" && e.DataSource != "users" && e.DataSource != "channels" && e.DataSource != "dynamic" {
```

Comment:
From the changes on this file I understand we can have a `select` that can have a dynamic data source, and a `dynamic_select`. Isn't that redundant? Is it intended?

Reply:
Appsform supports 'dynamic_select` but Interactive Dialog is type `select`, datasource `dynamic`. I actually did it both ways and missed removing 'dynamic_select` here.